cedar-policy-core/src/extensions/ipaddr.rs

```diff
@@ -86,89 +100,123 @@ impl IPAddr {
     /// Return true if this is a loopback address
     fn is_loopback(&self) -> bool {
         // Loopback addresses are "127.0.0.0/8" for IpV4 and "::1" for IpV6
-        // Unlike the implementation of `is_multicast`, we don't need to test prefix
-        // The reason for IpV6 is obvious: There's only one loopback address
-        // The reason for IpV4 is that provided the truncated ip address is a loopback address, its prefix cannot be less than 8 because otherwise its more significant byte cannot be 127
-        self.addr.is_loopback()
+        // If `addr` is a loopback address, its prefix is `127` or `1`
+        // We need to just make sure the prefix is greater than or equal to `8` or `128`
```

Comment:
I think this comment mixes two different meanings of the word "prefix"?

Reply:
Yes. I'll fix it.